vignettes/lintr.Rmd

```diff
@@ -9,6 +9,7 @@ vignette: >
 ---
 
 ```{r, include = FALSE}
+library(lintr)
```

Comment:
Is it standard practice in vignettes to silently load the package?

Reply:
Hmm, not super common:

https://github.com/search?q=org%3Acran%20path%3Avignettes%20%2Fopts_chunk%5B%24%5Dset%2F&type=code

I'd call this a false positive of `library_call_linter()` which should probably exempt antecedent `opts_chunk$set()` calls (though our `library(lintr)` probably _should_ appear higher in the vignette). I'll skip this change for now.